Formula/nuget.rb

```diff
@@ -0,0 +1,23 @@
+class Nuget < Formula
+  desc "the package manager for the Microsoft development platform including .NET"
+  homepage "https://www.nuget.org/"
+  url "https://dist.nuget.org/win-x86-commandline/v3.4.3/nuget.exe"
+  version "3.4.3"
+  sha256 "3b1ea72943968d7af6bacdb4f2f3a048a25afd14564ef1d8b1c041fdb09ebb0a"
+
+  bottle :unneeded
+
+  depends_on "mono" => :recommended
+
+  def install
+    libexec.install "nuget.exe"
+    (bin/"nuget").write <<-EOS.undent
+        #!/bin/bash
```

Comment:
Style nit but we tend to start the indent in `EOS` blocks two spaces in rather than four.